interfaces/builtin/wayland_test.go

```diff
@@ -190,6 +191,7 @@ KERNEL=="tty[0-9]*", TAG+="snap_wayland_app1"`)
 	spec = &udev.Specification{}
 	c.Assert(spec.AddPermanentSlot(s.iface, s.coreSlotInfo), IsNil)
 	c.Assert(spec.Snippets(), HasLen, 0)
+	c.Assert(spec.GetTriggeredSubsystems(), IsNil)
```

Comment:
I think in go you don't typically call functions `GetFoo` but `Foo`. This could become `spec.TriggeredSubsystems` instead.